src/actors/builtin/storage_miner/storage_miner_actor_code.go

```diff
@@ -23,11 +24,43 @@ import (
 // Actor methods
 ////////////////////////////////////////////////////////////////////////////////
 
-func (a *StorageMinerActorCode_I) GetWorkerVRFKey(rt Runtime) filcrypto.VRFPublicKey {
+func (a *StorageMinerActorCode_I) StageWorkerKeyChange(rt Runtime, key filcrypto.VRFPublicKey) (err error) {
```

Comment:
Actors never return errors, they only `rt.Abort()`

Reply:
duh. Thank you!